pshconfig/__init__.py

```diff
@@ -0,0 +1 @@
+from .config import *
```

Comment:
Since you are doing `import *`, you should include this in your `pshconfig/config.py` file:

```
__all__ = [
    "Config",
]
```

`__all__` is used to specify what names can be imported via `*`